src/interfaces/special-resolution-interfaces/special-resolution-interface.ts

```diff
@@ -9,4 +10,5 @@ export interface SpecialResolutionIF {
   nameRequest?: NameRequestIF,
   contactPoint?: ContactPointIF,
   courtOrder?: CourtOrderIF
+  cooperativeAssociationType: AssociationTypes
```

Comment:
I think I've found a way we can generate this interface from the JSON schema directly

removed the URLs.. removed the .json extension, used this: https://github.com/bcherny/json-schema-to-typescript

Reply:
Could you show this tomorrow at Developer Demo?

Reply:
Yes, here are some examples
[special_resolution.d.txt](https://github.com/bcgov/business-edit-ui/files/9294947/special_resolution.d.txt)
[alteration.d.txt](https://github.com/bcgov/business-edit-ui/files/9294948/alteration.d.txt)
:

Reply:
Nice! So when do the IFs get created -- are they available during implementation of a new feature?

Reply:
as long as someone updates the schema repo, and regenerate the types (preferably with a version number attached) I don't see why not